File: app/src/main/java/tech/tedycoding/sepatuku/ui/home/HomeActivity.java
package tech.tedycoding.sepatuku.ui.home;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.ImageView;

import tech.tedycoding.sepatuku.R;
import tech.tedycoding.sepatuku.ui.cart.CartActivity;
import tech.tedycoding.sepatuku.ui.categories.CategoriesActivity;

public class HomeActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_home);

        View viewToolbar = findViewById(R.id.layout_toolbar);
        ImageView menuCart = viewToolbar.findViewById(R.id.menu_cart);
        ImageView menuFilter = viewToolbar.findViewById(R.id.menu_filter);

        menuCart.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent goCart = new Intent(HomeActivity.this, CartActivity.class);
                startActivity(goCart);
            }
        });

        menuFilter.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                Intent goCart = new Intent(HomeActivity.this, CategoriesActivity.class);
                startActivity(goCart);
            }
        });
    }
}

File: app/src/main/java/tech/tedycoding/sepatuku/ui/home/HomeActivity.kt
package tech.tedycoding.sepatuku.ui.home

import android.content.Intent
import android.os.Bundle
import android.view.View
import android.widget.ImageView
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment
import kotlinx.android.synthetic.main.activity_home.*
import kotlinx.android.synthetic.main.layout_toolbar_home.*
import tech.tedycoding.sepatuku.R
import tech.tedycoding.sepatuku.ui.cart.CartActivity

class HomeActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_home)

        menu_cart.setOnClickListener {
            val goCart = Intent(this@HomeActivity, CartActivity::class.java)
            startActivity(goCart)
        }

//        menu_filter.setOnClickListener {
//            val goCart = Intent(this@HomeActivity, CategoriesActivity::class.java)
//            startActivity(goCart)
//        }
    }

}